mailpoet/RoboFile.php

```diff
@@ -28,6 +28,13 @@ public function install() {
       ->run();
   }
 
+  public function installJs() {
+    return $this->taskExecStack()
+      ->stopOnFail()
+      ->exec('cd .. && pnpm install --frozen-lockfile --prefer-offline')
+      ->run();
+  }
```

Comment:
This is a minor observation, but I guess now this could be reused in the install function above (e.g., `->addCode(...)`).